frontend/src/component/feature/FeatureView/FeatureOverview/ReleasePlan/ChangeRequest/ReleasePlanAddChangeRequestDialog.tsx

```diff
@@ -1,30 +1,45 @@
 import { Dialogue } from 'component/common/Dialogue/Dialogue';
 import useToast from 'hooks/useToast';
 import { styled, Button } from '@mui/material';
-import type { IReleasePlanTemplate } from 'interfaces/releasePlans';
+import type {
+    IReleasePlan,
+    IReleasePlanMilestone,
+    IReleasePlanTemplate,
+} from 'interfaces/releasePlans';
 import { useChangeRequestApi } from 'hooks/api/actions/useChangeRequestApi/useChangeRequestApi';
+import { usePendingChangeRequests } from 'hooks/api/getters/usePendingChangeRequests/usePendingChangeRequests';
 
 const StyledBoldSpan = styled('span')(({ theme }) => ({
     fontWeight: theme.typography.fontWeightBold,
 }));
 
 interface IReleasePlanAddChangeRequestDialogProps {
+    action: 'addReleasePlan' | 'deleteReleasePlan' | 'startMilestone';
```

Comment:
Is this needed now that we split them up?

Reply:
Strange, I removed these. why doesn't that show up?